ares/pce/system/system.hpp

```diff
@@ -38,7 +38,8 @@ struct System {
 extern System system;
 
 auto Model::PCEngine()    -> bool { return system.model() == System::Model::PCEngine;    }
-auto Model::PCEngineDuo() -> bool { return system.model() == System::Model::PCEngineDuo; }
+auto Model::PCEngineDuo() -> bool { return (system.model() == System::Model::PCEngineDuo) || (system.model() == System::Model::LaserActive); }
```

Comment:
nit: we'll probably want to clean this up at some point because, despite sharing functionality, PC Engine Duo and LaserActive are two different models.